modules/cudacodec/src/ffmpeg_video_source.cpp

```diff
@@ -192,14 +192,8 @@ void cv::cudacodec::detail::FFmpegVideoSource::updateFormat(const FormatInfo& vi
 
 bool cv::cudacodec::detail::FFmpegVideoSource::get(const int propertyId, double& propertyVal) const
 {
-    CV_Assert(videoCaptureParams.size() % 2 == 0);
```

Comment:
Are `CAP_PROP_OPEN_TIMEOUT_MSEC` and `CAP_PROP_READ_TIMEOUT_MSEC` going to be queryable from `VideoCapture` going forward?  If not should they be stored here so the information is available to the user of the `VideoReader` class?

Reply:
`CAP_PROP_OPEN_TIMEOUT_MSEC` and `CAP_PROP_READ_TIMEOUT_MSEC` are marked as open-only parameters of `VideoCapture`. It does not return value on get call. In case if open-only parameters are important for reader, I can restore open parameters handling.

Reply:
> It does not return value on get call. In case if open-only parameters are important for reader, I can restore open parameters handling.

I can't think of a specific reason for storing these two values but I also can't think of a reason why not if its just a simple case of restoring the parameter handling before querying `VideoCapture::get`.

Reply:
Restored back.